### ISSUE TITLE
Add contention graphs and open sql connections to metrics page in CC

### ISSUE THREAD

**Issue by xinhaoz** [cockroachdb/cockroach#73566]
As per https://github.com/cockroachdb/cockroach/issues/69799, the metrics page with some preliminary graphs was created for CC serverless. The following metrics need to be surfaced and the graphs should be added to the metrics page:

- [x] Open SQL Sessions - "The total number of open SQL sessions. This includes both active and idle sessions."
- [x] Transaction contention - "The total number of transactions that experienced contention."
- [x] SQL Statement Contention - "The total number of statements that experienced contention."
- [x] SQL Connection latency (90th and 99th percentile) - "The connection latency in nanoseconds between when the cluster receives a connection request and establishes the connection to the client, including authentication."
- [x] SQL Connection attempts - "The total number of connection attempts to the cluster".  (sql.new_conns)
- [ ] SQL Connection errors - "The total number of connection errors to the cluster."

Epic CC-5060

Jira issue: CRDB-11647

**Comment by Azhng** [comment]
Writing up a quick update here, CRDB currently doesn't emit the second `Transaction Contention` metrics. 

However, emitting the transaction contention metrics is currently blocked by a bug (#78675). (this is either a bug within a the trace analyzer or vectorized stats collector, still investigating).

**Comment by kevin-v-ngo** [comment]
Thanks for updating the thread @Azhng. I added the 3 Connection metrics as well which came up in a couple threads. They should be already available but if not, let me know and we can create a separate issue.

**Comment by Azhng** [comment]
I think the `SQL Connection errors` is not yet available.

**Comment by kevin-v-ngo** [comment]
@Azhng can you paste screenshots of the charts here? Adding @Annebirzin as FYI

**Comment by Azhng** [comment]
https://user-images.githubusercontent.com/9267198/163240962-83a60540-8230-42eb-8832-7e582b0144f4.mov

**Comment by Azhng** [comment]
Oops the issue got closed by accident.

**Comment by Annebirzin** [comment]
Nice! I had a few nits (mostly copy updates):

SQL Connection Latency
-  it looks like there are 2 lines represented but we are missing the bottom key to tell you what those lines mean (it would look similar to the SQL Statement Latency graph key)
- The y-axis shows `ms` twice 

SQL Statement Latency
- The y-axis shows `ms` twice 

SQL Open Sessions
- Update title to `Open SQL Sessions` (consistent with Open SQL Transactions)
- Update the y-axis to just `Sessions` to remove redundancy with the title and make it consistent with Open SQL Transactions

SQL Connection Attempts  
- Update the y-axis to `Connection Attempts` to simplify and remove redundancy with the title 

Full Scans
- For the y-axis, capitalize 'Scans'

Statement Errors
- For the y-axis, capitalize 'Errors'

SQL Statement Contention
- Update the y-axis to `Statements`  


General
- For graphs like SQL Connection Attempts, SQL Statment Errors, or SQL Statement Contention (among others), I'm not sure it makes sense to have fractions on the y-axis. For example, would it ever show 0.2 statements with contention? Could we make it so graphs with whole number counts always start at 1? (this might be a separate issue)

cc @kevin-v-ngo 
cc @stbof for any other copy feedback
